test/Microsoft.SqlTools.ServiceLayer.UnitTests/DisasterRecovery/SharedAccessSignatureCreatorTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Moq;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using Azure.Storage.Blobs;
+using Microsoft.SqlTools.ServiceLayer.AzureBlob;
+using Microsoft.SqlServer.Management.Smo;
+using Azure.Storage.Sas;
+
+namespace Microsoft.SqlTools.ServiceLayer.UnitTests.DisasterRecovery
+{
+    class SharedAccessSignatureCreatorTests
+    {
+
+        [Test]
```

Comment:
are we planning to add more tests for other code paths in this PR?

Reply:
I'm not sure we need more unit tests. However it would be nice to add some integration tests.

Reply:
I've managed to set up my environment for integration tests, and I added a backup integration test. We need to add an Environment variable for configuring azure blob connection parameters. I named the variable "AzureBlobSettingFileName", and it's a path to a configuration file. Config file has the following format:
```
{
	"BlobContainerUri" : "<test blob container uri>",
	"AccountKey" : "<test account key>",
	"AccountName" : "<test account name>"
}
```

I plan to add restore integration test too.

Reply:
I added restore integration test too and added ability to pass all Azure blob variables via environment variables.